ambry-server/src/main/java/com/github/ambry/server/AmbryServer.java

```diff
@@ -309,6 +312,14 @@ public void startup() throws InstantiationException {
                 http2ClientConfig, metrics, nettyMetrics, http2ServerMetrics, serverSecurityService);
         nettyHttp2Server = nioServerFactory.getNioServer();
         nettyHttp2Server.start();
+        Gauge<Long> nettyWorkerPendingTasks =
```

Comment:
Could this guage registration be done from inside of `StrorageServerNettyFactory`?